resources/general/videos/what-is-event-driven-architecture.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'The Many Meanings of Event-Driven Architecture • Martin Fowler • GOTO 2017'
 description: |
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras vitae iaculis nisl. Phasellus quis auctor lorem. Suspendisse potenti. Nunc semper dapibus molestie. Donec ultrices consectetur'
+      Towards the end of last year I attended a workshop with my colleagues in ThoughtWorks to discuss the nature of “event-driven” applications. Over the last few years we've been building lots of systems that make a lot of use of events, and they've been often praised, and often damned. Our North American office organized a summit, and ThoughtWorks senior developers from all over the world showed up to share ideas. The biggest outcome of the summit was recognizing that when people talk about “events”, they actually mean some quite different things. So we spent a lot of time trying to tease out what some useful patterns might be. This note is a brief summary of the main ones we identified.
```

Comment:
Not sure about this one. But this is the abstract Martin Fowler used.

Reply:
Yeah seems a bit long? 
![image](https://user-images.githubusercontent.com/3268013/153376516-d44c51cf-bce0-448e-9afd-f6de0aefae99.png)

Maybe we should try and reduce it abit? What do you think?

Reply:
Yes, I think it's a good idea to shorten it a bit. People don't want to read too much. I think concise information is better.

Reply:
What about:

> In this talk, Martin Fowler presents a summary of the most common patterns of EDA: Event Notification, Event-carried State Transfer, Event Sourcing, and CQRS.

Reply:
I think it's better now, I like this kind of description better. However, I am curious to know what others also think about this 🙂

Reply:
Yeah looks better, thank you @smoya !